browser/components/CLAEditor.tsx

```diff
@@ -190,67 +189,65 @@ export default class CLAEditor extends React.Component<Props, State> {
-              <datalist id="contact">
-                {this.state.data.display.poc.map(user => {
-                  return (<option key={user} value={user}/>);
-                })}
-              </datalist>
-              <br/>
-              <label>Date Signed</label>
-              <DatePicker name="dateSigned" dateFormat="MM/DD/YYYY" id="dateSigned" value={this.state.data.signed_date} onChange={this.handleDateChangeForSigned}/>
-              <br/>
-              <label>Date Approved</label>
-              <DatePicker name="dateApproved" id="dateApproved" value={this.state.data.approved_date} onChange={this.handleDateChangeForApproval}/>
-              <br/>
-              <label>Additional Notes</label>
-              <br/>
-              <textarea id="description" className="form-control" rows={3} placeholder={this.state.data.additional_notes} name="description" required></textarea>
+        <form id="contributions-form" onSubmit={this.handleSubmit}>
+          <div className="form-group">
+            <label>Project name</label>
+            <br/>
+            <input type="text" list="project_name" id="projectName" className="form-control" name="projectName" placeholder={this.state.data.project_name}/>
+            <datalist id="project_name">
+              {this.getOptionsProjectNames()}
+            </datalist>
+            <br/>
+            <label>Contributor[s]</label>
+            <br/>
+            <input type="text" className="form-control" id="contributorName" name="contributorName" placeholder={this.state.data.contributor_names}/><br/>
+            <label>Approver</label>
+            <br/>
+            <input type="text" list="approver_names" className="form-control" name="appName" id="appName" placeholder={this.state.data.approver_name}/>
+            <datalist id="approver_names">
+              {this.getOptionsApproverNames()}
+            </datalist>
+            <br/>
+            <label>Signatory</label>
+            <br/>
+            <input type="text"  list="signatory_name" className="form-control" name="sigName" id="sigName" placeholder={this.state.data.signatory_name}/>
+            <datalist id="signatory_name">
+              {this.state.data.display.signatory.map(user => {
+                return (<option key={user} value={user}/>);
+              })}
+            </datalist>
+            <br/>
+            <label>Point of Contact</label>
+            <br/>
+            <input type="text"  list="contact" className="form-control" id="contactName" name="contactName" placeholder={this.state.data.contact_name}/>
+            <datalist id="contact">
+              {this.state.data.display.poc.map(user => {
+                return (<option key={user} value={user}/>);
+              })}
+            </datalist>
+            <br/>
+            <label>Date Signed</label>
```

Comment:
Your date changes cause the following error to be thrown on submit with empty date:
error:  error: invalid input syntax for type date: ""
    at Connection.parseE (/home/local/ANT/jamesiri/code/oss-contribution-tracker/node_modules/pg/lib/connection.js:539:11)
    at Connection.parseMessage (/home/local/ANT/jamesiri/code/oss-contribution-tracker/node_modules/pg/lib/connection.js:366:17)
    at Socket.<anonymous> (/home/local/ANT/jamesiri/code/oss-contribution-tracker/node_modules/pg/lib/connection.js:105:22)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at Socket.Readable.push (_stream_readable.js:208:10)
    at TCP.onread (net.js:594:20

Reply:
Marking those as required.